packages/commonwealth/client/scripts/views/pages/why_commonwealth.tsx

```diff
@@ -56,7 +56,7 @@ class WhyCommonwealthPage extends ClassComponent {
               project's most important on-chain actions like staking and voting.
               Commonwealth has a growing list of integrations--including
               Ethereum, NEAR, Cosmos, Substrate, and popular DAO frameworks like
-              Moloch or Aragon.
+              or Aragon.
```

Comment:
Fix this copy.

Reply:
fixed(i think)

Reply:
nope, just remove the "or" lol

Reply:
"Ethereum, NEAR, Cosmos, Substrate, and popular DAO frameworks
              Aragon"

nope!

Reply:
My bad, fixed now 🤦